Rmarkdown/Oncology_Efficacy_Plots.Rmd

```diff
@@ -77,6 +103,31 @@ dose_record <- dose_record %>%
 
 dose_intensity <- dose_record[c("IDSHORT","COMB","ave_dose_intensity")]
 dose_intensity <- subset(dose_intensity, !duplicated(IDSHORT))
+
+#units and labels
+time_units_dataset     = "hours"
+time_units_plot        = "days"
+trtact_label           = "Dose"
+dose_label             = "Dose (mg)"
+time_label             = "Time(Days)"
+timemonth_label        = "Time (months)"
+sld_label              = "Percent Change in\nSum of Longest Diameters"
+sadose_label           = "ABC123 Dose (mg)"
+combodose_label        = "DEF456 Dose (mg)"
```

Comment:
combo_dose_label

Reply:
it's easier to read if you add "_" between different words

Reply:
done!